scripts/03_airdrop.js

```diff
@@ -1,11 +1,11 @@
 const { ethers } = require('hardhat');
 require('dotenv').config();
-const contract = require('../src/artifacts/contracts/KiftVans.sol/KiftVans.json');
+const contract = require('../src/artifacts/contracts/Kiftables.sol/Kiftables.json');
 const { testAddresses } = require('../config/config');
 const { ALCHEMY_API_KEY, PRIVATE_KEY, CONTRACT_ADDRESS, TEST_WALLET_OWNER } =
   process.env;
 const alchemyProvider = new ethers.providers.AlchemyProvider(
-  (network = 'maticmum'),
+  (network = 'rinkeby'),
```

Comment:
the `// airdrop mint is performed during deploy. just check balance` comment down below is no longer accurate, right?

Reply:
yup, need to clean up the tests now